storage/src/main/java/com/tencent/rss/storage/handler/impl/AbstractClientReadHandler.java

```diff
@@ -36,4 +37,12 @@ public ShuffleDataResult readShuffleData() {
   @Override
   public void close() {
   }
+
+  @Override
+  public void feedbackConsumedBlock(BufferSegment bs) {
```

Comment:
Why add these two interface and could you add comments if they are necessary.

Reply:
The handler only returns the segment, but does not know the actually consumed blocks.
In order to collect detailed information, the consumer should feedback this information, and each type of handler can design their rules.